src/ign.cc

```diff
@@ -131,7 +133,8 @@ extern "C" SDFORMAT_VISIBLE int cmdDescribe(const char *_version)
 }
 
 //////////////////////////////////////////////////
-extern "C" SDFORMAT_VISIBLE int cmdPrint(const char *_path)
+extern "C" SDFORMAT_VISIBLE int cmdPrint(const char *_path,
```

Comment:
Any thoughts?

Reply:
Would it be cleaner to create a new function, and only deprecate the older function in the next major release?

Reply:
sorry I missed this mention. This symbol is exported so that it can be called from our ruby script, but it is not linked from any external libraries. I think it is safe to make this change, and the ABI checker CI job hasn't complained